### PR TITLE
RealLifeGTAV Presence: update

### PR DESCRIPTION
**Pull request by N0chteil**
Added buttons, pages

Screenshot_228
Screenshot_226
Screenshot_227

### CODE REVIEW

**Review by Bas950 — APPROVED** [review]

File: websites/R/RealLifeGTAV/dist/metadata.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "forum.reallifegtav.de",
     "forum.rlv.link"
   ],
-  "version": "2.0.1",
+  "version": "2.1.0",
```
Comment (ririxidev) [inline websites/R/RealLifeGTAV/dist/metadata.json:18]: 2.0.0 xD

**Review by ririxidev — DISMISSED** [review]

File: websites/R/RealLifeGTAV/dist/metadata.json (hunk 1 above)
Comment (Bas950) [reply websites/R/RealLifeGTAV/dist/metadata.json]: what no? He did it correct? Now the version is 1 lower...

File: websites/R/RealLifeGTAV/dist/metadata.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "forum.reallifegtav.de",
     "forum.rlv.link"
   ],
-  "version": "2.0.1",
+  "version": "2.0.0",
```
Comment (StrikerFRFX) [inline websites/R/RealLifeGTAV/dist/metadata.json:18]: Change back to 2.1.0 as it was correct the first time

File: websites/R/RealLifeGTAV/presence.ts [hunk 3]
```diff
@@ -60,15 +60,15 @@ presence.on("UpdateData", async function () {
           set_showUsername,
           " | User: " + document.querySelector("#RPC_Username").textContent
         );
-      presenceData.smallImageKey = "reading";
+
```
Comment (StrikerFRFX) [inline websites/R/RealLifeGTAV/presence.ts:63]: Remove unnecessary newlines but also separate blocks of code

File: websites/R/RealLifeGTAV/presence.ts [hunk 4]
```diff
@@ -11,6 +11,7 @@ presence.on("UpdateData", async function () {
     },
     set_showUsername = await presence.getSetting("showUsername"),
     set_timeElapsed = await presence.getSetting("timeElapsed"),
+    set_showButtons = await presence.getSetting("showButtons"),
     urlpath = window.location.pathname.split("/");
   function ifSettingEnabled(setting, string) {
```
Comment (StrikerFRFX) [inline websites/R/RealLifeGTAV/presence.ts:16]: What? Just use a ternary operator in place of this wherever you need to check a setting is toggled... 
X48vo3BzLw https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

File: websites/R/RealLifeGTAV/presence.ts [hunk 5]
```diff
@@ -60,15 +60,15 @@ presence.on("UpdateData", async function () {
...
-      presenceData.smallImageKey = "reading";
+
     } else if (document.location.pathname.includes("/faq")) {
       presenceData.details =
         "FAQ" +
         ifSettingEnabled(
           set_showUsername,
           " | User: " + document.querySelector("#RPC_Username").textContent
         );
-      presenceData.smallImageKey = "reading";
```
Comment (StrikerFRFX) [inline websites/R/RealLifeGTAV/presence.ts:71]: Why remove smallImageKey?

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]
See my comments.

File: websites/R/RealLifeGTAV/presence.ts (hunk 5 above)
Comment (N0chteil) [reply websites/R/RealLifeGTAV/presence.ts:71]: I don't like small images

**Comment by N0chteil** [comment]
🙄Github...

File: websites/R/RealLifeGTAV/presence.ts [hunk 6]
```diff
@@ -11,115 +11,54 @@ presence.on("UpdateData", async function () {
...
       urlpath[2] != ""
     ) {
-      presenceData.smallImageKey = "downloading";
       presenceData.details =
-        "Startseite" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
+        "Startseite" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
```
Comment (Bas950) [inline websites/R/RealLifeGTAV/presence.ts:28]: is that correct?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/R/RealLifeGTAV/presence.ts (hunk 5 above)
Comment (StrikerFRFX) [reply websites/R/RealLifeGTAV/presence.ts:71]: You should use either choose to use it all the time in your file, or choose to never use it at all. Imo, you should use it but see if you can find some nice icons.

File: websites/R/RealLifeGTAV/presence.ts [hunk 7]
```diff
@@ -11,115 +11,54 @@ presence.on("UpdateData", async function () {
...
+        "FAQ" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
     } else if (document.location.pathname.includes("/jobs")) {
       presenceData.details =
-        "Jobs" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
-      presenceData.smallImageKey = "writing";
+        "Jobs" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
```
Comment (StrikerFRFX) [inline websites/R/RealLifeGTAV/presence.ts:43]: Line 43 and other related lines with `if()`, I never once said to change to an if statement. Use. A. Ternary. Operator. It lets you define a variable based on if it is true or false, which would be used in your case. Also good for Code Optimisation and shortening.

**Comment by N0chteil** [comment]
I'll create a new pull request because I broke something here 😂

**Pull request closed without merging**